### PR TITLE
Update homework 1 files

### PR DESCRIPTION
**Pull request by jlpalomino**
These changes update the files for homework 1.

### CODE REVIEW

**Comment by lwasser** [comment]
@jlpalomino  this assignment looks just fine to me. i'm going to jump over to the notebook next and have a look at what they will see in the notebook. What i think you should do is get a bit of feedback from them on the homework assignment in the second course survey. ...

**Comment by lwasser** [comment]
ok having a hard time finding the notebook. the one in the education repo seems to be a few days old and somewhat empty... but what i see here looks ok. if you have specific questions please let me know. otherwise let's see how the homework goes!!

File: _posts/courses/earth-analytics-bootcamp/00-course-overview/2018-07-16-co-04-homework-1.md [hunk 1]
```diff
@@ -21,7 +21,186 @@ sidebar:
...
 
+
+## Part 1 - Homework 1
+
+This component will test your `Shell`, `Jupyter Notebook` and `Git` skills from Days 1 to 3. 
+
+Complete the <a href="https://canvas.colorado.edu/courses/17063/quizzes/38134" target = "_blank">Homework 1 Quiz on CANVAS.</a> You need to log into CANVAS to see this quiz. This component contributes 50 points to your homework grade (out of a total possible points of 100).
+
+
+## Part 2 - Homework 1
```
Comment (lwasser) [inline _posts/courses/earth-analytics-bootcamp/00-course-overview/2018-07-16-co-04-homework-1.md]:
```
## Part 2 - Complete and Submit a Jupyter Notebook
The second part of this homework assignment will involve you working on a set of questions in a jupyter notebook. Open the file `name-here` in your xxxx repo. Then  do the following:
```
Be super clear about the steps and what they are doign. when i read this i'm not sure if i'm opening this from the repo i forked above??

File: _posts/courses/earth-analytics-bootcamp/00-course-overview/2018-07-16-co-04-homework-1.md [hunk 2]
```diff
@@ -21,7 +21,186 @@ sidebar:
...
+1. Which months have the lowest and the highest temperatures? How much of a difference is there between the lowest and highest temperatures?
+
+2. Given that Boulder, CO is in the Northern Hemisphere, are these results surprising? How might the results look different if you were plotting data from a location in the Southern Hemisphere? 
+
+3. Given the Boulder, CO is at a moderately high latitude (~40 degrees North), are these results surprising? How might the results look different if you were plotting data from a location closer to the equator? (Hint: think about the difference between the lowest and highest temperatures)
+
+To answer these questions, it can help to review the <a href="https://www.google.com/maps/place/Boulder,+CO/@18.3736306,-81.2965677,3.85z/data=!4m5!3m4!1s0x876b8d4e278dafd3:0xc8393b7ca01b8058!8m2!3d40.0149856!4d-105.2705456" target = "_blank">location of Boulder Colorado on Google Maps</a>. 
+
+
+## Submit Part 2 - Homework 1 Using Git/Github.com
```
Comment (lwasser) [inline _posts/courses/earth-analytics-bootcamp/00-course-overview/2018-07-16-co-04-homework-1.md]: This is a bit confusing 
how about 
## Part 3 - Submit Jupyter Notebook to Github xxx repo 

submit part 2 is confusing .

**Comment by lwasser** [comment]
i think you are in good shape. do you have a rubric yet for this homework?
It would be good to have this on the website and you can add it monday or tuesday. but normally i have a rubric available when i provide the homework. not a huge deal. edit this... merge it. then don't forget to PR your notebook files for all of these things! And to update the homework repo IF it needs to be updated. i'm not clear on whether you plan to have a scripted notebook for them or not.

**Comment by jlpalomino** [comment]
@lwasser Thanks for your comments. They were very helpful for making the instructions more clear. I've also updated the homework repository to include the correct notebook for the students.

**Pull request merged**